src/components/Navbar/Navbar.tsx

```diff
@@ -31,7 +31,7 @@ export const Navbar = () => {
         <Logo variant={isLearnPage ? 'learn' : 'default'} />
       </LogoLink>
       {!isLearnPage ? <NavbarTabs navbarPageReload={isLoyaltyPage} /> : null}
-      <NavbarButtons redirectToLearn={isLearnPage} />
```

Comment:
@dennyscode on the `loyaltyPage`, we should offer a wallet connection, not a redirection to the "/"

Reply:
Thanks! Fixed now